atom_core/src/atom_core/dataset_io.py

```diff
@@ -580,6 +580,16 @@ def filterSensorsFromDataset(dataset, args):
 
     return dataset
 
+def deleteCollectionFromDataset(dataset,collection_number):
+    # Use the filterCollectionsFromDataset instead. Check #948.
+    # Leaving this function here for potential future use only.
+ 
+    # Converting int 90 to '090'
+    collection_key = generateCollectionKey(collection_number)
+    print(f'Deleting {collection_key}')
+    del dataset['collections'][collection_key]
```

Comment:
Again, this is a different way to filter collections, and I think there should only be one

Reply:
Removed this function. It was leftover from a mistake I made back then.